src/mpc/results.jl

```diff
@@ -8,6 +8,8 @@ MPC Scenarios will return a results Dict with the following keys:
 - `ElectricUtility`
 - `PV`
 - `Generator`
+- `Electrolyzer`
+- `FuelCell`
 """
 function mpc_results(m::JuMP.AbstractModel, p::MPCInputs; _n="")
```

Comment:
doesn't look like we're handling multinode in MPC so probably shouldn't allow this as an arg